packages/editor/src/components/post-actions/index.js

```diff
@@ -29,26 +29,38 @@ const {
 
 export default function PostActions( { onActionPerformed, buttonProps } ) {
 	const [ isActionsMenuOpen, setIsActionsMenuOpen ] = useState( false );
-	const { item, postType } = useSelect( ( select ) => {
+	const { item, permissions, postType } = useSelect( ( select ) => {
 		const { getCurrentPostType, getCurrentPostId } = select( editorStore );
-		const { getEditedEntityRecord } = select( coreStore );
+		const { getEditedEntityRecord, getEntityRecordPermissions } = unlock(
+			select( coreStore )
+		);
 		const _postType = getCurrentPostType();
+		const _id = getCurrentPostId();
 		return {
-			item: getEditedEntityRecord(
+			item: getEditedEntityRecord( 'postType', _postType, _id ),
+			permissions: getEntityRecordPermissions(
```

Comment:
It looks like we also need to memoize `getEntityRecordPermissions`. Currently, it triggers a warning in the post editor.

## Screenshot
![CleanShot 2024-07-30 at 14 03 18](https://github.com/user-attachments/assets/7a8d9cc5-529e-42ae-a6bb-b1d29a1f0790)

Reply:
I'll handle this one. I know what's the issue.

Reply:
Resolved by #64091.